api/bag_vrrefinementsdescriptor.cpp

```diff
@@ -54,7 +56,7 @@ std::shared_ptr<VRRefinementsDescriptor> VRRefinementsDescriptor::create(
     int compressionLevel)
 {
     return std::shared_ptr<VRRefinementsDescriptor>(
-        new VRRefinementsDescriptor{dataset.getNextId(), chunkSize,
+        new VRRefinementsDescriptor{dataset.getNextId(), 1, 1, chunkSize,

```

Comment:
@brian-r-calder I changed this to 1 row and 1 column to get the existing tests to pass, but it seems like this should start out as 0, 0 and only be changed if a refinement is added. What do you think?